spec/passfort/http_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Passfort::Http do
+  shared_examples "an error response" do
```

Comment:
Not a huge fan of having one shared example that uses another shared example. It's a bit confusing...